src/openai_responses/_utils/serde.py

```diff
@@ -15,7 +15,7 @@ def json_loads(b: bytes) -> Any:
 
 def model_dict(m: BaseModel) -> dict[str, Any]:
```

Comment:
Just update these function signatures and where you used `model_dict` in moderations and we're good 👍